30-remote-site-setup.sh

```diff
@@ -78,11 +79,19 @@ grep '^OSG_GRID="/cvmfs/oasis.opensciencegrid.org/osg-software/osg-wn-client' \
      /var/lib/osg/job-environment*.conf > /dev/null 2>&1
 cvmfs_wn_client=$?
 
+override_opts=""
 for ruser in $users; do
     setup_ssh_config
     [[ $cvmfs_wn_client -eq 0 ]] || setup_endpoints_ini
+    if [[ -n $OVERRIDE_DIR ]]; then
```

Comment:
While it's not broken as you have it, i think this new `if` block really belongs above the `for` loop; just to be executed once.